src/components/structures/RoomView.js

```diff
@@ -180,6 +177,11 @@ module.exports = React.createClass({
         roomProm.then((room) => {
             this._calculatePeekRules(room);
             return this._initTimeline(this.props);
+        }).catch(() => {
```

Comment:
I don't like this, because there are about 1000 other reasons this could have failed.

Why don't we just make the initialState of `timelineLoading` be false?